src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,193 @@
+package parser;
+
+// Import Commands
+import command.AddCommand;
+import command.Command;
+import command.DeleteCommand;
+import command.ExitCommand;
+import command.FindCommand;
+import command.ListCommand;
+import command.UpdateCommand;
+
+// Import Luna Exceptions
+import exception.LunaException;
+import exception.LunaInvalidCommandException;
+import exception.LunaInvalidDateException;
+import exception.LunaInvalidDescriptionException;
+import exception.LunaInvalidIndexException;
+
+// Import other exceptions
+import java.time.format.DateTimeParseException;
+
+// Import Tasks;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.TaskList;
+import task.Todo;
+
+// Imports for dates
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deals with making sense of the user command.
+ *
+ * @author fannyjian
+ */
+public class Parser {
+
+    /**
+     * Takes in a string of user command and creates a new instance of Command
+     * according to the keyword.
+     *
+     * @param cmd String of command to process.
+     * @return Command corresponding to the user's instruction.
+     * @throws LunaException.
+     */
+    public static Command parse(String cmd) throws LunaException {
+        String[] cmdSplit = cmd.split(" ", 2);
+        Command c;
```

Comment:
Perhaps you can consider using more intuitive variable names like "command", "splitCommand" etc, and not use c (unless for iterator variables)